scripts/test/deployAll/system-test-instance.properties

```diff
@@ -65,193 +65,23 @@ sleeper.optional.stacks=SystemTestStack,CompactionStack,GarbageCollectorStack,In
 # Whether to keep the sleeper table bucket, Dynamo tables related to tables and query results bucket or not.
 sleeper.retain.infra.after.destroy=false
 
-# The maximum number of concurrent compaction tasks to run
-sleeper.compaction.max.concurrent.tasks=300
-
-# Partitions with more than the following number of records in will be split
-sleeper.default.partition.splitting.threshold=100000000
-
 # How often partition splitting should happen
 sleeper.partition.splitting.period.minutes=10
```

Comment:
Maybe this should be removed so that the default is used?

Reply:
As this is a test scenario we want a more frequent period than the default which is 30 minutes. Suggest changing this to 2 minutes and adding a comment to say this is deliberately a lot less than the default as this is a test scenario.